modules/gql.yml

```diff
@@ -0,0 +1,17 @@
+name: gql
```

Comment:
`gql` slug is too generic to be listed. I would rather reserve it for a potential built-in or ad-hoc feature of Nuxt. Would be nice if we could use a more relevant name like `graphql-request` or `graphql-codegen` (either that is the main intend integration of nuxt module).